.ci/cmake.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
You should make these two files executable for Travis not to choke on them. That is: `chmod u+x cmake.sh`